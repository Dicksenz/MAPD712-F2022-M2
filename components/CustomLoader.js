import React from "react";

import { StyleSheet, Text, View, ActivityIndicator } from "react-native";

const CustomLoader = () => {
  return (
    <View style={styles.container}>
      <ActivityIndicator size="large" color="#347174" />
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    backgroundColor: "white",
    display: "flex",
    justifyContent: "center",
    height: "100%",
    alignItems: "center",
  },
  textStyle: {
    fontSize: 20,
  },
});

export default CustomLoader;
