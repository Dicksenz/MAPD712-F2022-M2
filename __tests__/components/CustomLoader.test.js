import React from "react";
import renderer from "react-test-renderer";
import CustomLoader from "../../components/CustomLoader";

test("Check if CustomLoader component renders correctly", () => {
  const tree = renderer.create(<CustomLoader />).toJSON();
  expect(tree).toMatchSnapshot();
});
