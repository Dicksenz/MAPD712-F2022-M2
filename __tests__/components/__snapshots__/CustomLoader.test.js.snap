// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if CustomLoader component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    color="#347174"
    size="large"
  />
</View>
`;
